lib/cocina/models/barcode.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module Cocina
+  module Models
+    Barcode = BusinessBarcode | LaneMedicalBarcode | CatkeyBarcode | StandardBarcode
+  end
+end
```

Comment:
Here's what the new generated union types look like.

Reply:
schweet!